app/helpers/helpers.rb

```diff
@@ -0,0 +1,12 @@
+require 'sinatra/base'
+
```

Comment:
This code looks like it was intended to serve the purpose of the `current_user` and `logged_in?` methods that are already implemented in your ApplicationController. I would recommend refactoring to remove any unused code.